package.json

```diff
@@ -37,19 +37,21 @@
     "homepage": "https://github.com/universalbasket/node-framework#readme",
     "devDependencies": {
         "@types/chalk": "^2.2.0",
+        "@types/jsonwebtoken": "^8.5.0",
         "@types/mocha": "^5.2.6",
         "@types/node": "^12.7.3",
         "@types/supertest": "^2.0.7",
-        "@typescript-eslint/eslint-plugin": "^3.0.1",
-        "@typescript-eslint/parser": "^3.0.1",
-        "eslint": "^7.1.0",
-        "eslint-config-recommended": "^4.0.0",
+        "@typescript-eslint/eslint-plugin": "^3.3.0",
+        "@typescript-eslint/parser": "^3.3.0",
+        "eslint": "^6.8.0",
```

Comment:
some of the peer dependencies do not support `^7.x` so it generates noise, so I downgraded to 6 for now..